### ISSUE TITLE
Handling Cases for Kakadu config. issues?

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/figgy#97]
While these are be local to my development environment, I would imagine that they might be encountered by those unfamiliar to the code base.

Upon attempting to ingest compressed images in the TIFF (using the release of Kakadu referenced on https://github.com/pulibrary/plum within the macOS operating system), the following error is reported by Sidekiq:

```
Exit code: pid 25562 exit 255
Error message: Kakadu Error:
The simple TIFF file reader in this demo application can only read uncompressed
TIFF files.  This has nothing to do with Kakadu itself, of course.  If you
would like to read compressed TIFF files, however, it should be sufficient to
re-compile this application with the symbol "KDU_INCLUDE_TIFF" defined, and
link it against the public-domain LIBTIFF library.
```

I can provide some documentation for this, but might there be some way of updating the UV for cases such as these?  (A dialog with "error" is rendered).

**Comment by escowles** [comment]
Document limitations of Kakadu (not supporting compressed TIFFs)

**Comment by eliotjordan** [comment]
@jrgriffiniii Figgy can handle compressed tiffs now, can this issue be closed?

**Comment by jrgriffiniii** [comment]
Resolved by #1351
